.travis.yml

```diff
@@ -14,247 +14,254 @@
-                               netgen                           \
-                               netgen-headers                   \
-                               oce-draw                         \
-                               pyside-tools                     \
-                               python-dev                       \
-                               python-pyside                    \
-                               python-matplotlib                \
-                               python3-dev                      \
-                               python3-pyside                   \
-                               python3-matplotlib               \
-                               qt4-dev-tools                    \
-                               qt4-qmake                        \
-                               libqt4-opengl-dev                \
-                               shiboken                         \
-                               swig                             \
-                               libvtk6-dev                      \
-                               libmed-dev                       \
-                               libmedc-dev			\
-                               asciidoc
+       brew tap cartr/qt4
 
-       #Install Eigen 3.3.3 to reduce compiler warnings
-       curl -L "http://bitbucket.org/eigen/eigen/get/3.3.3.tar.gz" | tar xvz && cd eigen-*
-       mkdir build && cd build
-       cmake -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -DCMAKE_BUILD_TYPE=Release -DCMAKE_VERBOSE_MAKEFILE=ON ..
-       sudo make -j2 install
-       cd ${TRAVIS_BUILD_DIR}
+       brew install --verbose --only-dependencies freecad --with-qt4 --with-packaging-utils
+       brew install --verbose --only-dependencies freecad --with-qt4 --with-packaging-utils #Ensure all dependencies are satisfied
 
-       #Install ccache to reduce successive build times (use 3.2.5+ to avoid spurious warnings)
-       curl -L "https://codeload.github.com/ccache/ccache/tar.gz/v3.2.5" | tar xvz && cd ccache-*
-       ./autogen.sh && ./configure CC=gcc && sudo make install
-       cd ${TRAVIS_BUILD_DIR}
+    elif [ "${QT}" == "Qt5" ]; then
+       # Qt5: Replace Qt4 in ports-cache with Qt5, if necessary
+       brew ls --versions cartr/qt4/qt > /dev/null && \
+          brew uninstall --ignore-dependencies --force cartr/qt4/shiboken@1.2 cartr/qt4/pyside@1.2 cartr/qt4/pyside-tools@1.2 cartr/qt4/qt
 
-       #Patch the system - there is a bug related to invalid location of libs on ubuntu 12.04
-       sudo ln -s /usr/lib/x86_64-linux-gnu/ /usr/lib/i386-linux-gnu
-       sudo find /usr/lib -name libpq.so -exec ln -s {} /usr/lib/libpq.so ';'
-       export DISPLAY=:99.0
-       sh -e /etc/init.d/xvfb start
+       brew install --verbose --only-dependencies freecad --with-packaging-utils
+       brew install --verbose --only-dependencies freecad --with-packaging-utils #Ensure all dependencies are satisfied
```

Comment:
why is this line duplicated?

Reply:
Because this is still unresolved https://github.com/FreeCAD/homebrew-freecad/issues/38